### DISCUSSION TITLE
如何使用pixiv插件搜索/How do I search on Pixiv after I have already installed the booru-pixiv plugin?

### DISCUSSION THREAD

**Discussion by xanthichi** [koishijs/koishi-plugin-booru#231] [Q&A]
我已经装好了booru-pixiv和booru两个插件并且都已经启用，refreshtoken也配置好了，可是我搜索他总是报错（booru-pixiv插件启用后左面的依然是灰色的？）
I have already installed and enabled both the 'booru-pixiv' and 'booru' plugins, and I have also configured the 'refresh token', but when I search, it always reports an error (after enabling the 'booru-pixiv' plugin, the left side is still gray?)
![image](https://github.com/koishijs/koishi-plugin-booru/assets/135406843/5626e9ac-f95a-4b48-a157-fba6945d9eec)
当我去沙盒尝逝使用，他直接回复找不到，日志如下报错
When I try to use it in the sandbox, it directly replies that it cannot find it, and the error is reported in the log as follows.
![image](https://github.com/koishijs/koishi-plugin-booru/assets/135406843/776a81f1-97fa-4f82-8404-6c0cba1d97ec)

**Comment by MaikoTan** [comment]
![image](https://github.com/user-attachments/assets/93d4e55c-e5fd-48d4-8df1-ea06e8616be3)

You might misunderstand the meaning of the option `proxyAgent`, it should be set an HTTP or SOCKS general proxy server, not `pixiv.cat`, a REVERSE proxy. (And you even lack the `https://` prefix...)

你可能误解了 `proxyAgent` 选项的含义，它应该设置为一个通用的 HTTP 或 SOCKS 代理服务器，而不是 `pixiv.cat` 这种 **反向代理**。(你甚至缺少了 `https://` 前缀……)

Since you are using Chinese, I assume you are living in China. So you need a VPN or something like `Clash` that serves as an HTTP server in your local. VPN should be set in the system config, not Koishi itself or even the plugin. As for `Clash`, it will open an HTTP port such as `7890`, then you can just write down `http://localhost:7890/` here.

既然你使用汉语，我假定你住在中国。因此你需要一个虚拟专用网络或其他像是 `Clash` 这样服务为一个 HTTP 服务器的东西在你的本地。VPN 应该在系统设置中设置，不是 Koishi 也不是插件。至于 `Clash`，它会开放一个 HTTP 埠像是 `7890`，然后你可以在这里写下 `http://localhost:7890`。
